edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -49,11 +49,29 @@ var edx = edx || {};
        });
   };
 
+  function createWorker(url) {
+    var blob, blobBuilder, blobUrl, url;
+    var importScript = "importScripts('" + location.origin + url + "');";
+
+    try {
+      blob = new Blob([importScript], { "type": 'application/javascript' });
+    } catch (e) {
+      //older browsers do not support the Blob constructor, so fallback to the deprecated BlobBuilder
+      //interface in those cases
+      blobBuilder = new (window.BlobBuilder || window.WebKitBlobBuilder || window.MozBlobBuilder)();
+      blobBuilder.append(importScript);
+      blob = blobBuilder.getBlob('application/javascript');
+    }
+
+    url = window.URL || window.webkitURL;
+    blobUrl = url.createObjectURL(blob);
```

Comment:
Similarly here I don't think we need the `webkitURL` fallback https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL